migrations/migrate-duplicate-emails.sql

```diff
@@ -0,0 +1,52 @@
+--------------------------------------------------------------------------------
+-- For each relation, find all records tied to a learn user that also have a
+-- github account with the same email, re-associate the records with the github
+-- account
+--------------------------------------------------------------------------------
+UPDATE membership
+  SET "user" = guser.id
+  FROM "user" luser
+  JOIN "user" guser
+    ON guser.email = luser.email
+  WHERE luser.id = membership.user
+    AND guser.plugin = 'github'
```

Comment:
Ah, right - I read this query backwards. I was looking at the tables in the "FROM" clause thinking those tables were being updated.

Reading through this again, everything makes sense.